docs/multi-organization-guide.md

```diff
@@ -0,0 +1,338 @@
+
+ADF applications often contain environment / AWS Organization stage specific
+configuration files. In order to allow AWS CodeBuild to select the proper
+configuration context for an application, the environment / AWS Organization stage
+context needs to be made available. A simple pattern to solve this problem is the
+introduction of the SSM parameter `adf/org/stage` in the buildspec file of the
+application. This SSM parameter will be auto-created by ADF. The following snippet
+shows the header of such a `codebuild.yaml` file.
+
+```yaml
+env:
+  parameter-store:
+    ADF_ORG_STAGE: "/adf/org/stage"
+[...]
+```
+
+This environment variable can then be used to drive decision/deployment logic
+within any of the subsequent build commands/actions.
+
+Some scenarios which could require organization specific context include:
+
+- Deriving the default log level based on the organization stage for
+a specific CDK application
+- Appending the Stage name to AWS resource names having a requirement to be
+  both deterministic as well as globally unique
+(whilst being deployed into multiple organizations).
+- Selecting a config file from a config folder with the following files:
+  - `config-dev.yaml`
+  - `config-int.yaml`
+  - `config-prod.yaml`
+- Copy a specific `params` folder to use for the given organization context,
+  before `generate_params.py` is invoked:
+  `cp -r ./org_specific/${ADF_ORG_STAGE}/params ./params`
+
+### 5. Customize the Base IAM Roles Per AWS Organization
+
+ADF Supports Bootstrapping Baseline CloudFormation Stacks to all AWS Accounts
+when they first join an AWS Organization and centrally governing the subsequent
+lifecycle of those stacks. [More information on bootstrapping accounts can be
+found in the admin guide](admin-guide.md#bootstrapping-accounts).
+
+These Baseline Templates are typically used for setting up the default IAM
+roles and Policies necessary for the foundations of an ADF Based Enterprise
+Landing Zone.
+
+In guidance with AWS Security Guidelines and `Least Privilege Access Principles`,
+it is recommended to reduce the scope of any IAM Policy to the minimum required
+Actions, Principals and Resource Scope necessary.
+
+To customize the scope of which resources or principals are permitted within the
+IAM Policies of the baseline templates CFN Mapping fields can be utilized based
+on the `Org Stage` SSM Parameter. As shown below:
+
+```yaml
+Parameters:
+  OrgStage:
+    Type: "AWS::SSM::Parameter::Value<String>"
+    Description: Org Stage
+    Default: /adf/org/stage
+# At the time this Stack is deployed, the FinOps Account ID SSM Parameter doesn't
+# exist, so we derive it from mapping it to the org stage
+Mappings:
+  # Usage:!FindInMap [OrgStageBasedPropertyMap, !Ref OrgStage, FinOpsAccountId]
+  OrgStageBasedPropertyMap:
+    dev:
+      FinOpsAccountId: 1234567891012 # Dev Org
+    int:
+      FinOpsAccountId: 1234567891013 # Int Org
+    prod:
+      FinOpsAccountId: 1234567891014 # Prod Org
```

Comment:
In accordance with our (Contributing Guide](https://github.com/awslabs/aws-deployment-framework/blob/master/CONTRIBUTING.md#use-of-examples) I suggest changing these to account ids in the example list of account IDs, so our code scanners don't flag this later on:

```suggestion
    dev:
      FinOpsAccountId: 111111111111 # Dev Org
    int:
      FinOpsAccountId: 222222222222 # Int Org
    prod:
      FinOpsAccountId: 333333333333 # Prod Org
```